src/main/java/com/google/api/codegen/viewmodel/ClientMethodType.java

```diff
@@ -32,7 +32,7 @@
   // PHP
   OptionalArrayMethod,
   PagedOptionalArrayMethod,
-  OperationOptionalArrayMethod,
+  LongRunningOptionalArrayMethod,
```

Comment:
And this is only used in PHP and Python, not in the other dynamic languages?

I would be in favor of changing the comment so that this line ends with `// Python, PHP` to indicate that.

Reply:
Done